.pipelines/singletenancy/azure-cni-overlay/azure-cni-overlay-e2e-step-template.yaml

```diff
@@ -161,3 +161,87 @@ steps:
     name: "Cleanupartifactdir"
     displayName: "Cleanup artifact dir"
     condition: always()
+
+  - task: AzureCLI@2
+    inputs:
+      azureSubscription: $(AZURE_TEST_AGENT_SERVICE_CONNECTION)
+      scriptLocation: "inlineScript"
+      scriptType: "bash"
+      addSpnToEnvironment: true
+      inlineScript: |
+        make -C ./hack/aks windows-nodepool-up AZCLI=az SUB=$(SUB_AZURE_NETWORK_AGENT_BUILD_VALIDATIONS) CLUSTER=${{ parameters.clusterName }}
+        echo "Windows node are successfully added to v4 Overlay Cluster"
+        kubectl cluster-info
+        kubectl get node
+        kubectl get po -owide -A
+    name: "Add_Windows_Node"
+    displayName: "Add windows node on v4 overlay cluster"
+
+  - task: AzureCLI@2
+    inputs:
+      azureSubscription: $(AZURE_TEST_AGENT_SERVICE_CONNECTION)
+      scriptLocation: "inlineScript"
+      scriptType: "bash"
+      addSpnToEnvironment: true
+      inlineScript: |
+        set -e
+        echo "Deploying windows CNS on v4 overlay cluster"
+        export WINDOWS_CNS_VERSION=$(make cns-version)
+        envsubst '${WINDOWS_CNS_VERSION}' < test/integration/manifests/cns/daemonset-windows.yaml | kubectl apply -f -
+        echo "wait for CNS pods deployment ready"
+        kubectl rollout status ds azure-cns-win -n kube-system --timeout 5m
+        kubectl get po -owide -A
+        kubectl describe ds azure-cns-win -n kube-system | grep Image
+        echo "Upload CNI"
+        dropgz_version=$(make cni-dropgz-test-version)
+        export DROP_GZ_URL=$( make cni-dropgz-test-image-name-and-tag OS='windows' ARCH='amd64' OS_VERSION='ltsc2022' CNI_DROPGZ_VERSION=${dropgz_version})
+        envsubst < ./test/integration/manifests/cni/cni-installer-v1-windows.yaml | kubectl apply -f -
+    name: "Deploy_CNS_CNI"
+    displayName: "Deploy CNS and upload CNI on v4Overlay cluster"
+
+  - script: |
+      kubectl taint nodes aksnpwin000000  node.cloudprovider.kubernetes.io/uninitialized=true:NoSchedule-
+      kubectl taint nodes aksnpwin000001  node.cloudprovider.kubernetes.io/uninitialized=true:NoSchedule-
+      cd test/integration/load
+      echo "IPv4 Overlay Windows control plane CNS validation test"
+      sudo go test -timeout 30m -tags load -run ^TestLoad$ -os=windows
```

Comment:
I think we should just run all these load tests, and make load test suite have a way of installing CNS daemonset, in go-lang code, similar to how `make test-integration` does it

We could even have 1 central go-lang code, that installs CNS and Dropgz, so that both test suites (`test/ingetration` and `test/integration/load` can use the same code)